src/main/java/nextstep/ladder/Direction.java

```diff
@@ -0,0 +1,73 @@
+package nextstep.ladder;
+
+import java.util.Objects;
+
+import static java.lang.Boolean.FALSE;
+
+public class Direction {
+    private final boolean left;
+    private final boolean right;
+
+    private Direction(boolean left, boolean right) {
+        if (left && right) {
+            throw new IllegalStateException("유효하지 않은 상태 값 입니다.");
+        }
```

Comment:
피드백 반영 잘해주셨네요 👍
아래처럼 조금 더 자세히 메시지를 적어주면 좋겠네요 😄
```suggestion
        if (left && right) {
            throw new IllegalStateException("같은 라인에 인접하게 생성될 수 없습니다.");
        }
```